### PR TITLE
chore: update github issue templates; add docs/troubleshooting

### PR DESCRIPTION
**Pull request by andrewseguin**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.md [hunk 1]
```diff
@@ -1,21 +1,31 @@
...
+What behavior did you actually see?
 
 
-#### Is there anything else we should know?
+#### Environment
+
+- Angular:
+- CDK/Material:
+- Browser(s):
+- Operation System (e.g. Windows, OSX, Ubuntu):
```
Comment (crisbeto) [inline .github/ISSUE_TEMPLATE/bug.md]: Operation system -> Operating system

File: .github/ISSUE_TEMPLATE/docs.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
```
Comment (crisbeto) [inline .github/ISSUE_TEMPLATE/docs.md]: It seems weird to have a table in here, or does GitHub do something special with it?

File: .github/ISSUE_TEMPLATE/bug.md [hunk 3]
```diff
@@ -1,21 +1,31 @@
...
+What behavior did you actually see?
 
 
-#### Is there anything else we should know?
+#### Environment
+
+- Angular:
+- CDK/Material:
+- Browser(s):
+- Operating System (e.g. Windows, OSX, Ubuntu):
```
Comment (devversion) [inline .github/ISSUE_TEMPLATE/bug.md]: Very nitpicky: macOS instead?

File: .github/ISSUE_TEMPLATE/docs.md (hunk 2 above)
Comment (devversion) [reply .github/ISSUE_TEMPLATE/docs.md]: GitHub automatically picks the information up when showing the available issue templates.

Comment (andrewseguin) [reply .github/ISSUE_TEMPLATE/docs.md]: Yeah - it's a format that GitHub uses to get metadata https://help.github.com/en/articles/manually-creating-a-single-issue-template-for-your-repository (see step 5)

File: .github/ISSUE_TEMPLATE/bug.md (hunk 3 above)
Comment (andrewseguin) [reply .github/ISSUE_TEMPLATE/bug.md]: Not nitpicky at all - good catch

**Pull request merged**